src/ILCompiler.Compiler/src/Compiler/CompilerTypeSystemContext.cs

```diff
@@ -127,9 +129,10 @@ protected override ModuleData CreateValueFromKey(string key)
         }
         private SimpleNameHashtable _simpleNameHashtable = new SimpleNameHashtable();
 
-        public CompilerTypeSystemContext(TargetDetails details)
+        public CompilerTypeSystemContext(TargetDetails details, bool multifileCompilation)
```

Comment:
The new methods in this file deal more with compilation than type system - they should really be named something like "IsPartOfCompilationGroup" than "IsDefined".

Revert changes to this file and move the methods to Compilation.cs?